mockdb.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+    availableItems: [
+        {
+            user: "N. Maier",
```

Comment:
I would suggest Fantasy Usernames like "Morpheus" or "Gandalf" to underline the anonymity of the concept.

Reply:
Sounds good to me. Anonymity is a good point, I didn't think of that.

Reply:
I fixed this in #21 .